images/openwisp_openvpn/openvpn_utils.sh

```
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+get_redis_value() {
+	local key="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)